server/server.ts

```diff
@@ -3,51 +3,50 @@ import http from 'http'
 import https from 'https'
 import path from 'path'
 
-import { Web3Provider } from '@ethersproject/providers'
 import cors from 'cors'
-import express from 'express' 
+import express from 'express'
 import MiniRouteLoader from 'mini-route-loader'
 
-import AppHelper from '../lib/app-helper' 
+import AppHelper from '../lib/app-helper'
 import FileHelper from '../lib/file-helper'
 import MongoInterface from '../lib/mongo-interface'
 
 import ApiController from './controllers/api-controller'
 
 require('dotenv').config()
+const bodyParser = require('body-parser')
 
 const routes = FileHelper.readJSONFile('./server/config/routes.json')
 
- 
 export default class WebServer {
   server: https.Server | http.Server | undefined
 
   async start(serverConfig: any): Promise<void> {
-     
-
     const dbName = AppHelper.getDbName()
 
     const mongoInterface = new MongoInterface()
-    await mongoInterface.init(dbName) 
-   
+    await mongoInterface.init(dbName)
 
     const apiController = new ApiController(mongoInterface)
 
     const app = express()
     const apiPort = serverConfig.port ? serverConfig.port : 3000
 
     app.use(cors())
-    app.use(express.json())
-
-    if (serverConfig.useHTTPS == true) {
-      this.server = https.createServer({
-        cert: fs.readFileSync('/home/andy/deploy/cert/starflask.com.pem'),
-        key: fs.readFileSync('/home/andy/deploy/cert/starflask.com.key'),
+    
+    app.use(
+      bodyParser.json({
+        type: '*/*',
+        verify: (req:any, res:any, buf:Buffer) => {
+           req.rawBody = buf
+           return true
+        },
+        limit: '10mb', // https://stackoverflow.com/a/19965089/1165441
       })
```

Comment:
hmm curious what this does?

Reply:
I added a comment about this.  It is necessary as per the bloom devs due to how they send their webhook data